oeps/best-practices/oep-0067-bp-tools-and-technology.rst

```diff
@@ -0,0 +1,342 @@
+OEP-67: Tools and Technology Standards
+######################################
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - Link to the doc in the following format::
+
+        :doc:`OEP-67 <oep-0067-bp-tools-and-technology>`
+
+   * - Title
+     - Tools and Technology Standards
+   * - Last Modified
+     - 2023-09-05
+   * - Authors
+     - Feanil Patel <feanil@axim.org>
+   * - Arbiter
+     - TBD
+   * - Status
+     - Accepted
+   * - Type
+     - Best Practice
+   * - Created
+     - 2023-09-05
+   * - Review Period
+     - 2023-09-05 - 2023-10-xx
+   * - Resolution
+     - <links to any discussions where the final status was decided>
+   * - References
+     - :doc:`oep-0011-bp-FED-technology`
+
+Abstract
+********
+
+The Open edX Platform is built using many 1st and 3rd party tools and technologies.
+This document covers the current recommendations for the tools and technologies
+that should be used.
+
+Motivation
+**********
+
+The Open edX platform has a lot of 1st and 3rd party tools and technology that it uses.
+The problems we're trying to solve are:
+
+    * It is difficult to know what the current best practice for a given area.
+
+    * It is unclear how to go about the discussion and decision making process
+      to change a best practice.
+
+:doc:`OEP-11 <oep-0011-bp-FED-technology>` serves as a great example for the
+usefulness of this kind of OEP but limits itself to be relevant only to
+frontend technologies.  This OEP would supersede :doc:`OEP-11
+<oep-0011-bp-FED-technology>` and encompass not only frontend technolog but any
+technology that:
+
+    * Is considered a best practice for the Open edX Platform
+
+    * Is relevant to a large subset of the Open edX Platform
+
+Specification
+*************
+
+.. note:: We'll fill this out before the OEP lands but leaving mostly blank for initial feedback.
```

Comment:
Consider ordering the spec so that the import stuff is generally closer to the top. Seeing Codecov as the first item in the spec feels a little off, given how minor of a choice it is in the grand scheme of things. Maybe having the subsection order be [Python, JS, General] would be more conducive to that.

Reply:
Thinking more about this, I think it would be fine to Move general to the bottom so the order would be [JS, Python, General]  let me know if others have opinions on this and I can make a decisions at some point based on feedback.

Reply:
@feanil I do like that idea, but it's not blocking for me.